website/docs/r/oracle_cloud_vm_cluster.html.markdown

```diff
@@ -81,73 +81,75 @@ resource "azurerm_oracle_cloud_vm_cluster" "example" {
 
 The following arguments are supported:
 
-* `cloud_exadata_infrastructure_id` - (Required) The [OCID](https://docs.cloud.oracle.com/iaas/Content/General/Concepts/identifiers.htm) of the Cloud Exadata infrastructure.
+* `cloud_exadata_infrastructure_id` - (Required) The [OCID](https://docs.cloud.oracle.com/iaas/Content/General/Concepts/identifiers.htm) of the Cloud Exadata infrastructure. Changing this forces a new Cloud VM Cluster to be created.
 
-* `cpu_core_count` - (Required) The number of CPU cores enabled on the Cloud VM Cluster.
+* `cpu_core_count` - (Required) The number of CPU cores enabled on the Cloud VM Cluster. Changing this forces a new Cloud VM Cluster to be created.
 
-* `db_servers` - (Required) The list of DB servers.
+* `db_servers` - (Required) The list of DB servers. Changing this forces a new Cloud VM Cluster to be created.
 
-* `display_name` - (Required) The user-friendly name for the Cloud VM Cluster. The name does not need to be unique..
+* `display_name` - (Required) The user-friendly name for the Cloud VM Cluster. Changing this forces a new Cloud VM Cluster to be created. The name does not need to be unique.
 
-* `gi_version` - (Required) A valid Oracle Grid Infrastructure (GI) software version.
+* `gi_version` - (Required) A valid Oracle Grid Infrastructure (GI) software version. Changing this forces a new Cloud VM Cluster to be created.
 
-* `hostname` - (Required) The hostname for the Cloud VM Cluster without suffix.
+* `hostname` - (Required) The hostname for the Cloud VM Cluster without suffix. Changing this forces a new Cloud VM Cluster to be created.
 
-* `license_model` - (Required) The Oracle license model that applies to the Cloud VM Cluster, either `BringYourOwnLicense` or `LicenseIncluded`.
+* `hostname_actual` - The hostname for the Cloud VM Cluster with suffix. Changing this forces a new Cloud VM Cluster to be created.
 
```

Comment:
looks like this is already covered in the Attributes reference below as it's `Computed` only property?
```suggestion
```

Reply:
Good point, applied your suggestion thanks!